javascript-forms/form-values.js

```diff
@@ -0,0 +1,14 @@
+var $form = document.querySelector('#contact-form');
+
+var userInfo = {};
```

Comment:
You should move this variable inside of the function definition. It doesn't need to be used outside that function, so keeping it inside the function is the better choice